de.metas.document.archive/de.metas.document.archive.base/src/main/java/de/metas/document/archive/process/ExportArchivePDF.java

```diff
@@ -51,9 +55,24 @@ protected String doIt()
 		final IArchiveBL archiveBL = Services.get(IArchiveBL.class);
 		final byte[] data = archiveBL.getBinaryData(archive);
 		final String contentType = archiveBL.getContentType(archive);
-		final String filename = null;
+		final String filename = String.valueOf(archive.getRecord_ID());
```

Comment:
hmm, is this OK? Is it opened nicely on webui?

Reply:
Yes. I mean is just used as the pdf name the id. (the easiest way to have a name)
![image](https://user-images.githubusercontent.com/15378036/31897136-29f993c0-b81e-11e7-95f3-d58506da5003.png)
)

But, in the end I think we should use the archive name. 
But that is broken now.
I will do a follow up task. Is related to pdf printing, I think.

Reply:
thanks, also pls check what happens when u try to save it (ctrl+s). What's the suggested filename?
I guess it would be your number (without pdf extension)...

Reply:
Yes. Is the number and no extension. ('All files' where is the extension)
And after you save it and you try to open, windows is suggesting to open it with adobe acrobat reader.